### PR TITLE
Remove react-abc library

### PR DESCRIPTION
**Pull request by Vanamo**

### CODE REVIEW

File: src/components/music/MusicSheet.js [hunk 1]
```diff
@@ -76,29 +78,28 @@ class MusicSheet extends React.Component {
 
     const renderNotes = this.state.renderNotes
     const renderSound = this.state.renderSound
-    const notation = this.props.notation
 
     return (
       <>
-        {renderNotes && this.renderNotation(notation)}
-        {renderSound && this.renderPlayButton(notation)}
-        {!renderSound && <div className="emptyPlayButton" />}
```
Comment (AuroraLS3) [inline src/components/music/MusicSheet.js]: This needs to be added back

**Review by AuroraLS3 — CHANGES_REQUESTED** [review]

File: src/components/music/MusicSheet.js (hunk 1 above)
Comment (Vanamo) [reply src/components/music/MusicSheet.js]: ok

File: package.json [hunk 2]
```diff
@@ -36,12 +36,12 @@
     "gatsby-transformer-remark": "^2.1.19",
     "interactjs": "^1.3.4",
     "jquery": "^3.4.1",
+    "midi": "github:paulrosen/MIDI.js#abcjs",
     "moment": "^2.24.0",
     "moocfi-quizzes": "0.1.15",
     "pdf-slideshow": "^0.1.4",
     "prismjs": "^1.15.0",
     "react": "^16.7.0",
-    "react-abc": "github:nigoshh/react-abc-mooc#build",
```
Comment (nigoshh) [inline package.json:44]: success!

**Review by AuroraLS3 — APPROVED** [review]

**Review by nigoshh — APPROVED** [review]
Great!

**Pull request merged**